pandas/types/validate.py

```diff
@@ -0,0 +1,7 @@
+from pandas.types.common import is_bool
+
+def _validate_bool_type(value, arg_name):
```

Comment:
I would maybe call this `_validate_bool_kwarg` to make it clearer that it is specifically to validate a keyword